Jenkinsfile

```diff
@@ -0,0 +1,25 @@
+pipeline {
+  agent {
+     label 'coreos && !rkt-fly'
+  }
+
+  stages {
+    stage('build-image') {
+      when {
+        branch 'master'
+      }
+      environment {
+        QUAY = credentials('174f0169-803f-4840-8e07-b8bfa204f5c6')
+      }
+      steps {
+        sh """
+        cat /etc/os-release
+        docker version
+        docker login -u=$QUAY_USR -p=$QUAY_PSW quay.io
+        export CNI_VERSION=v0.5.2
```

Comment:
nit: Could this be sourced from a single location? Having it stored multiple times means it could get out of sync